sap/aibus/dar/client/dar_session.py

```diff
@@ -158,6 +158,17 @@ def post_data_to_endpoint(
         self._check_status_code(response, url)
         return response
 
+    def post_to_url(self, url: str, payload: dict, retry: bool = False) -> Response:
+        """
```

Comment:
Probably we should add some documentation here

Reply:
Added